part-2-server/createPoemFiles.js

```diff
@@ -0,0 +1,48 @@
+var poem = [ 'Take', 'this', 'kiss', 'upon', 'the',
+  'brow!','And,','in','parting','from',
+  'you','now,','Thus','much','let','me',
+  'avow','You','are','not','wrong,',
+  'who','deem','That','my','days',
+  'have','been','a','dream;',
+  'Yet','if','hope','has','flown','away',
+  'In','a','night,','or','in','a','day,',
+  'In','a','vision,','or','in','none,',
+  'Is','it','therefore','the','less','gone?',
+  'All','that','we','see','or','seem','Is',
+  'but','a','dream','within','a','dream.',
+  'I','stand','amid','the','roar',
+  'Of','a','surf-tormented','shore,',
+  'And','I','hold','within','my','hand','Grains',
+  'of','the','golden','sand--',
+  'How','few!','yet','how','they','creep',
+  'Through','my','fingers','to','the','deep,',
+  'While','I','weep--while','I','weep!','O',
+  'God!','can','I','not','grasp','Them','with',
+  'a','tighter','clasp?',
+  'O','God!','can','I','not','save',
+  'One','from','the','pitiless','wave?',
+  'Is','all','that','we','see','or','seem',
+  'But','a','dream','within','a','dream?' ]
+
+  var http = require('http');
+  var fs = require('fs');
+
+
+  var server = http.createServer(function(request,response){
+    var url  = request.url;
+
+    for(var i = 0; i < poem.length; i++){
+      fs.writeFile('poem-'+ i +'.txt', poem[i]);
+    };
+
+    if(url === '/poem-' + i ){
+      fs.readFile('poem-' i +'.txt'function(error,data){
+        if(error) throw error;
+        console.log(data.toString());
+      }) 
+      response.write(data.toString());
+    } else {
+        console.log('<h1> 404</h1><br><p>page not found</p>');
+    }
+  })
+  server.listen(3000);
```

Comment:
Could actually put this in your other server.js file. Generally you only need one server